KVKCalendar/Classes/TimelineView.swift

```diff
@@ -396,6 +396,10 @@ final class TimelineView: UIView, CompareEventDateProtocol {
     }
     
     private func scrollToCurrentTime(startHour: Int) {
+        guard style.timeline.enableAutoScroll else {
+            return
+        }
+        
         guard isEnabledAutoScroll else {
             isEnabledAutoScroll = true
             return
```

Comment:
@kvyatkovskys Line 405 - 409, "scrollView.setContentOffset(.zero, animated: true)", it scrolls to top

Reply:
yep, we can try this 
`guard style.timeline.scrollToCurrentHour else { return }
        
  guard let time = getTimelineLabel(hour: Date().hour) else {
          scrollView.setContentOffset(.zero, animated: true)
          return
   }` 
instead line 399 - 401

to use one property

Reply:
Ok yes, indeed, it would be better. I commit the new changes :)